controllers/question.js

```diff
@@ -13,6 +13,16 @@ const getQuestions = async (req, res) => {
   res.status(200).json(questions)
 }
 
+const deleteQuestion = async (req, res) => {
+  const userId = req.user.id
+  const { id } = req.params
+
+  await offerService.deleteQuestion(id, userId)
```

Comment:
Here need to use questionService instead of offerService

Reply:
Fixed